scipy/stats/tests/test_distributions.py

```diff
@@ -97,6 +97,67 @@ def test_all_distributions():
         yield check_distribution, dist, args, alpha
 
 
+def check_rvs_broadcast(distfunc, distname, args, loc, scale, shape):
+    sample = distfunc.rvs(*args, loc=loc, scale=scale)
+    assert_equal(sample.shape, shape, "%s: rvs failed to broadcast" % distname)
+
+
+def test_all_rvs_broadcast_shape():
+    # This test only checks the *shape* of the value returned by rvs().
+    for dist in dists:
```

Comment:
I'm moving the tests over to `test_continuous_basic.py` and `test_discrete_basic.py`, where they will use the lists `distcont` and `distdiscrete`.

> (I actually wonder why this is so [...])

I do that a lot when working with the distributions code. :)